.github/workflows/main.yml

```diff
@@ -44,6 +44,11 @@ jobs:
         with:
           path: target
           key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
+      - name: Run clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all-features
```

Comment:
I added this to the existing run only because I wasn't sure whether the caching would work otherwise.

Reply:
I'm unsure of whether this is the best way to add it.